bin/polly.py

```diff
@@ -309,19 +314,24 @@ def PositiveInt(string):
     cdir, args.verbosity, args.discard, args.tail, polly_toolchain
 )
 
+if args.cmake:
+    CMAKE_BIN = args.cmake
+else:
+    CMAKE_BIN = 'cmake'
+
 if os.name == 'nt':
   # Windows
-  detail.call.call(['where', 'cmake'], logging)
+  detail.call.call(['where', CMAKE_BIN], logging)
```

Comment:
I wasn't too sure what you meant by this, so I added the check just before this if block